lms/djangoapps/grades/tests/test_signals.py

```diff
@@ -251,3 +256,37 @@ def test_disconnect_manager_bad_arg(self):
         with self.assertRaises(ValueError):
             with disconnect_submissions_signal_receiver(PROBLEM_RAW_SCORE_CHANGED):
                 pass
+
+
+@ddt.ddt
+class UserPartitionChangeHandlerTest(TestCase):
+    """
```

Comment:
You can probably nix this docstring - the class only consists of one method, and that method's docstring says basically the same thing.